### DISCUSSION TITLE
Auto setup function

### DISCUSSION THREAD

**Discussion by PulseDoo** [fhdm-dev/scoppy#130] [General]
Hello,
it would be possible in one of the future releases to add an "_Auto setup_" button\function to the app in order to facilitate the configuration of the various parameters (e.g. sampling frequency, buffer length, trigger level, etc...) in particular when the nature of the connected signal is unknown (usually at the initial stage)?
If this feature is already present, could you please kindly tell me how to use it?

Thanks in advance.
Kind regards

**Comment by fhdm-dev** [comment]
That feature doesn't yet exist but it is on my to-do list :)

**Comment by PulseDoo** [comment]
Hello,
thank you so much for the feedback, I look forward to the time when it will be implemented.
Kind regards
